### PR TITLE
Cleanup: upgrade deprecations

### PR DESCRIPTION
**Pull request by thijskh**
This attempts to resolve most of #990.

### CODE REVIEW

File: webapp/src/Doctrine/HashPasswordSubscriber.php [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 use App\Entity\User;
 use Doctrine\Common\EventSubscriber;
-use Doctrine\Common\Persistence\Event\LifecycleEventArgs;
+use Doctrine\ORM\Event\LifecycleEventArgs;
```
Comment (nickygerritsen) [inline webapp/src/Doctrine/HashPasswordSubscriber.php:7]: I think we’re not using this import anymore?

**Review by nickygerritsen — APPROVED** [review]

File: webapp/src/Doctrine/HashPasswordSubscriber.php (hunk 1 above)
Comment (thijskh) [reply webapp/src/Doctrine/HashPasswordSubscriber.php:7]: We are using it in `public function preUpdate(PreUpdateEventArgs $args)`

Comment (nickygerritsen) [reply webapp/src/Doctrine/HashPasswordSubscriber.php:7]: That is PreUpdateEventArgs, not LifecycleEventArgs?

Comment (thijskh) [reply webapp/src/Doctrine/HashPasswordSubscriber.php:7]: *We are using it in `public function prePersist(LifecycleEventArgs $args)`

Comment (nickygerritsen) [reply webapp/src/Doctrine/HashPasswordSubscriber.php:7]: Now I see it 🤓

**Review by nickygerritsen — APPROVED** [review]

**Pull request merged**